src/angular-fontawesome.js

```diff
@@ -2,7 +2,7 @@ angular.module('picardy.fontawesome', [])
 	.directive('fa', function () {
 		return {
 			restrict: 'E',
-			template: '<i class="fa"></i>',
+			template: '<span class="fa" aria-hidden="true"></span>',
```

Comment:
I decided to 1) switch to using `<span>`s, which seems to be a happier path for accessibility. I'm going to bump a major version when we get done with this and a few more things to avoid breaking the world here.